src/components/Concentrator/ConcentratorVaultTable.tsx

```diff
@@ -71,12 +72,19 @@ export const ConcentratorVaultTable: FC<ConcentratorVaultTableProps> = ({
 }
 
 type ConcentratorVaultRowProps = {
-  concentratorTargetAsset: ConcentratorTargetAsset
+  concentratorTargetAsset: TargetAsset
   vaultAssetAddress: Address
   vaultType: VaultType
 }
 
 const ConcentratorVaultRow: FC<ConcentratorVaultRowProps> = (props) => {
-  const { data: vaultProps } = useConcentratorVault(props)
-  return <VaultRow {...vaultProps} />
+  const concentrator = useConcentrator(props)
+  if (!concentrator.data || !concentrator.data?.vault.type) return null
+  return (
+    <VaultRow
+      asset={concentrator.data.vault.asset}
+      type={concentrator.data.vault.type}
+      vaultAddress={concentrator.data.ybTokenAddress}
```

Comment:
Including `vaultAddress` allows us to pass any type of vault to a row without requiring registry requests later. This is crucial to making concentrators work.